tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2958,6 +2958,36 @@ public function userListsTheActivitiesForFolderOfSpaceWithDepthOrLimitUsingTheGr
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Then the activities should be in the following order:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theActivitiesShouldBeInTheFollowingOrder(TableNode $table): void {
+		$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent();
+		$activities = $responseBody->value;
+		$expectedOrder = [];
+		foreach ($table->getHash() as $row) {
+			$expectedOrder[] = [
+				'resource' => $row['resource'],
+				'message' => $row['message']
+			];
+		}
```

Comment:
this should not be necessary. please use `$table->getHash()` directly
```suggestion
foreach ($table->getHash() as $property => $expectedValue) {
```

Reply:
build activities list from `$activities` instead

Reply:
> build activities list from `$activities` instead

Couldn't get you on this one. Can you elaborate?